meta/saiserialize.cpp

```diff
@@ -84,6 +84,12 @@ sai_status_t transfer_list(
     {
         transfer_primitive(src_element.count, dst_element.count);
 
+        if (src_element.list == NULL && src_element.count > 0)
```

Comment:
Sorry, I thought the wrong direction. So move if-block before previous statement? So that there is no corrupted dst_element when throw?